.changeset/odd-wasps-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': minor
+---
+
+Added tabs component
```

Comment:
For changelogs its better to use the same approach as for commits if to follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). And it is a too poor description of a change.
IMO, its better to write like this:
> Add `STabsPanel` and `STab` components.